client/extensions/woocommerce/app/shipping/shipping-methods/local-pickup.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import FormCurrencyInput from 'components/forms/form-currency-input';
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormLabel from 'components/forms/form-label';
+import FormFieldSet from 'components/forms/form-fieldset';
+import Tooltip from 'components/tooltip';
+
+class LocalPickupMethod extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			showTooltip: false,
+			...props
+		};
+
+		this.showTooltip = this.showTooltip.bind( this );
+		this.hideTooltip = this.hideTooltip.bind( this );
+	}
+
+	showTooltip() {
+		this.setState( { showTooltip: true } );
+	}
+
+	hideTooltip() {
+		this.setState( { showTooltip: false } );
+	}
+
+	render() {
+		const __ = i18n.translate;
+
+		return (
+			<div className="shipping-methods__method-container shipping-methods__local-pickup">
+				<FormFieldSet>
+					<FormLabel>{ __( 'How much will you charge for local pickup?' ) }</FormLabel>
+					<FormCurrencyInput currencySymbolPrefix="$" value={ this.state.price } />
+				</FormFieldSet>
+				<FormFieldSet>
+					<FormCheckbox checked={ this.state.taxable } className="shipping-methods__local-pickup-taxable" />
+					{ __( 'Taxable' ) }
+					<span
+						className="shipping-methods__local-pickup-taxable-help"
+						ref="taxableHelp"
+						onMouseEnter={ this.showTooltip }
+						onMouseLeave={ this.hideTooltip } >
+					<Gridicon icon="help-outline" size={ 18 } />
+					</span>
+					<Tooltip
+						isVisible={ this.state.showTooltip }
+						context={ this.refs && this.refs.taxableHelp }
+						className="shipping-methods__local-pickup-taxable-tooltip is-dialog-visible"
+						position="top">
+						Taxable explanation
```

Comment:
We don't have a tooltip in core :) 

@jameskoster - I'm wondering if we should we only have this option in Taxes setup as a global option.  cc: our resident tax expert @allendav.